File: c#/shapes/Circle.cs
namespace Canvas;

public class Circle : Shape
{
    public int Radius { get; set; }

    public Circle(int radius)
    {
        Type = "circle";
        Radius = radius;
    }

    public override int CalArea(int borderThickness)
    {
        return (22 / 7) * (Radius + borderThickness) * (Radius + borderThickness);
    }

}

File: java/src/shapes/Circle.java
package shapes;

public class Circle extends Shape {

    public int radius;

    public Circle(int radius, int borderThickness) {
        this.radius = radius;
        this.borderThickness = borderThickness;
    }

    public int calculateArea(boolean withBorder) {
        if (withBorder) {
            return (22 / 7) * (radius + borderThickness) * (radius + borderThickness);
        }
        return (22 / 7) * radius * radius;
    }
}
